src/ROUTES.js

```diff
@@ -21,8 +21,10 @@ export default {
     REPORT,
     REPORT_WITH_ID: 'r/:reportID',
     getReportRoute: reportID => `r/${reportID}`,
-    IOU_REQUEST: 'iou/request',
-    IOU_BILL: 'iou/split',
+    IOU_REQUEST: 'iou/request/:reportID',
+    getIouRequestRoute: reportID => `iou/request/${reportID}`,
+    IOU_BILL: 'iou/split/:reportID',
+    getIouSplitRoute: reportID => `iou/split/${reportID}`,
```

Comment:
Could you leave in the old generic routes (we currently use these for testing IOU flow). I've temporarily hidden the global menu options you added previously, but once this feature is complete we'll be reenabling them.

Reply:
Sorry I wasn't clearer here, but could you leave the dynamic routes in too? (so include both the old and new routes :))

Reply:
@Julesssss sorry I'm still a little confused, do you mean leave the new routes but comment them out like the menu options on the report compose component? or do you mean leave the function routes?

also since reportID is optional, it should be backwards compatible if the new routes are left here.

Reply:
> do you mean leave the new routes but comment them out like the menu options on the report compose component? or do you mean leave the function routes?

No need to comment out, I wanted to include both new and old routes like so: 
```
    IOU_REQUEST: 'iou/request',
    IOU_REQUEST: 'iou/request/:reportID',
    getIouRequestRoute: reportID => `iou/request/${reportID}`,
    IOU_BILL: 'iou/split',
    IOU_BILL: 'iou/split/:reportID',
    getIouSplitRoute: reportID => `iou/split/${reportID}`,
```

> also since reportID is optional, it should be backwards compatible if the new routes are left here.

Ah, I see. That's why the old routes were originally removed? Because the old routes are redundant? If so, then the original commit was fine 🤦